callautomation-live-transcription/README.md

```diff
@@ -25,32 +25,21 @@ It accepts an incoming call from a phone number, performs DTMF recognition, and
 3. cd into the `callautomation-live-transcription` folder.
 4. From the root of the above folder, and with node installed, run `npm install`
 
-### Setup and host ngrok
-
-You can run multiple tunnels on ngrok by changing ngrok.yml file as follows:
-
-1. Open the ngrok.yml file from a powershell using the command ngrok config edit
-2. Update the ngrok.yml file as follows:
-    authtoken: xxxxxxxxxxxxxxxxxxxxxxxxxx
-    version: "2"
-    region: us
-    tunnels:
-    first:
-        addr: 8080
-        proto: http 
-        host_header: localhost:8080
-    second:
-        proto: http
-        addr: 5001
-        host_header: localhost:5001
-NOTE: Make sure the "addr:" field has only the port number, not the localhost url.
-3. Start all ngrok tunnels configured using the following command on a powershell - ngrok start --all
-4. Once you have setup the websocket server, note down the the ngrok url on your server's port as the websocket url in this application for incoming call scenario. Just replace the https:// with wss:// and update in the .env file.
 
 ### Add a Managed Identity to the ACS Resource that connects to the Cognitive Services Resource
 
 Follow the instructions in the [documentation](https://learn.microsoft.com/en-us/azure/communication-services/concepts/call-automation/azure-communication-services-azure-cognitive-services-integration).
 
+### Setup and host your Azure DevTunnel
+
+[Azure DevTunnels](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/get-started?tabs=windows) is an Azure service that enables you to share local web services hosted on the internet. Use the commands below to connect your local development environment to the public internet. This creates a tunnel with a persistent endpoint URL and which allows anonymous access. We will then use this endpoint to notify your application of calling events from the ACS Call Automation service.
+
+```bash
+devtunnel create --allow-anonymous
+devtunnel port create -p 8080
+devtunnel host
+```
+
```

Comment:
Please test and have the correlation id's in the description

Reply:
callConnectionId=1a004580-22bc-4f81-a85a-a3f9a23019ef, 
correlationId=d56162c1-2a38-405c-9767-2e7f89488d20